ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Services/TaskService.cs

```diff
@@ -59,9 +59,10 @@ public Task DeleteAsync(Guid id, CancellationToken cancellationToken)
             return _repository.DeleteAsync(id, cancellationToken);
         }
 
-        public async Task<TaskDTO> UpdateAsync(TaskDTO taskDto, CancellationToken cancellationToken)
+        public async Task<TaskDTO> UpdateAsync(UpdateTaskDTO updateTaskDto, CancellationToken cancellationToken)
         {
             cancellationToken.ThrowIfCancellationRequested();
+            var taskDto = _mapper.Map<TaskDTO>(updateTaskDto);
```

Comment:
Have you checked response of this endpoint ? Which project name for example is in response? Idea is to return **full** data about updated task

Reply:
See create method

Reply:
Please launch the app and check everything before the creating of PR. I see that correct code will not work at all, you should fix it